packages/mobx-binder-dayjs/src/conversion/DayjsConverter.ts

```diff
@@ -18,8 +18,13 @@ export class DayjsConverter implements Converter<BinderValidationResult, string
         return typeof formats === 'string' ? formats : formats[0]
     }
 
+    private isNotEmpty(value: string) {
+        const valueWithoutSpace = value.replace(/ /g, '');
+        return valueWithoutSpace.length > 2
```

Comment:
the check might not match specific needs of a date format. So implementing this via a filter might be better.

Reply:
ah, yes, you are right, we can implement a converter in SSS / specific project.